dashboard/dashboard.py

```diff
@@ -34,19 +109,27 @@ def report_download_page() -> None:
 def subscribe_page() -> None:
     """
     Creates subscribe page where users can subscribe
-    to daily summary PDF reports and/or alerts about
+    to daily summary PDF reports and / or alerts about
     trends and genres via email.
     """
     st.title("Subscribe Page")
-    st.write("Subscribe to updates and alerts below.")
+    st.write("Subscribe to daily reports and (optionally) alerts below.")
 
     # Email input
     email = st.text_input("Enter your email address:")
 
-    # Checkboxes for user subscriptions
-    subscribe_daily = st.checkbox("Subscribe to daily reports")
+    # Checkbox for user alerts
     subscribe_alerts = st.checkbox("Subscribe to alerts")
 
+    genres_dict = get_genres_from_db()
+    genre_names = list(genres_dict.keys())
+
+    # Multi-select for genres
+    selected_genres = st.multiselect(
+        "Select genres you want to receive updates about:",
+        genre_names
+    )
+
     # Submit button
     if st.button("Submit"):
         if not email:
```

Comment:
Maybe this error message should be "Email cannot be empty!" to differentiate between the errors?